pymc/distributions/continuous.py

```diff
@@ -2439,6 +2476,10 @@ def __new__(cls, name, nu, **kwargs):
     def dist(cls, nu, **kwargs):
         return Gamma.dist(alpha=nu / 2, beta=1 / 2, **kwargs)
 
+    def icdf(value, nu):
```

Comment:
Not needed. This Class just returns a Gamma variable so the icdf will work already like this.

Reply:
Very interesting!! Removed that! Thanks for pointing this out. 
Fixed in https://github.com/pymc-devs/pymc/pull/6845/commits/be24e627a698659037db8cb8affb9c82ad24d2ee